src/stores/applets.js

```diff
@@ -178,23 +144,17 @@ export const useAppletsStore = defineStore('applets', () => {
    * @param {string} [options.loadedMessage] message to display when loaded
    * @param {string} [options.errorMessage] message to display on error
    */
```

Comment:
Could you add some good doc to the docstring on what form `alerts` should/could, like it has for `options`? If someone new used this repo and wanted to get those errors reported, it would be quite tricky to figure out how without digging into the source code in CbAppletTarget.vue for an example.